syllabus/1-Cryptography/6-Advanced_Signatures-slides.md

```diff
@@ -36,8 +36,10 @@ Certificate transparency: [explanation](https://certificate.transparency.dev/how
 
 ### Certificates in Web3
 
-We are building systems that don't have a "Certificate Authority".<br/>
-But we can still use certificates in some niche instances.
+We are building systems that do not have a centralized "Certificate Authority".<br/>
+But we can still use certificates:
+ - Certified by a distributed authority.
```

Comment:
I think it is very important to get into the nuance on this topic if you mention it at this point. The main reason is that with a certificate authority, any and all certificates can be checked (largely) offline.[^1] However, in a distributed authority, there _can be no single key_. 

For instance, If you want to trust something signed by the validators of a blockchain in a trustless way, you need to sync the whole chain up to the current state in some way, and then make sure that you get signatures from a secure majority of the current validators. Or make sure that the multisignature address (as that's much more likely than individual sigs) has been confirmed on the chain in some way, etc.

 It's much more complex to use a certificate from a distributed authority than a CA.

I do like the other changes though, I agree niche was a bit excessive. 

[^1]: Not technically entirely true, due to things like checking databases for revocation certificates, checking the certificate transparency databases, etc. But fundamentally there is an offline chain of trust from the CA's hardcoded keys to the key providing them to you.